02-Calling-an-API/api-server.js

```diff
@@ -0,0 +1,44 @@
+const express = require("express");
+const cors = require("cors");
+const morgan = require("morgan");
+const helmet = require("helmet");
+const jwt = require("express-jwt");
+const jwksRsa = require("jwks-rsa");
+const { join } = require("path");
+const authConfig = require("./src/auth_config.json");
+
+const app = express();
+
+const port = process.env.API_PORT || 3001;
+const appPort = process.env.SERVER_PORT || 3000;
+
+if (!authConfig.domain || !authConfig.audience) {
+  throw new Error(
+    "Please make sure that auth_config.json is in place and populated"
+  );
+}
+
+app.use(morgan("dev"));
+app.use(helmet());
+app.use(cors({ origin: `http://localhost:${appPort}` }));
```

Comment:
This is the new line, the rest was moved from the app server (`server.js`)

Reply:
Let's make this configurable through the `auth_config.json` file but default to `http://localhost:${appPort}` if the configuration is not present. People may want to use this sample as a base and then potentially deploy it to a domain.

When we do a bigger iteration on this sample, let's make better use of environment variables as it will make deployment much easier - but that's for another day.